ui/initial_state.py

```diff
@@ -50,3 +53,9 @@ def write(state):
     write_state['shutting_down']     = False
     with open(state_file, 'w') as fh:
         pickle.dump(frozendict(write_state), fh)
+
+if __name__ == '__main__':
```

Comment:
This is quite ugly, why not import the module and use `initial_state.read()` direct when you need it?